File: app/src/main/java/com/example/a18/path/parabola/ParabolaActivity.java
package com.example.a18.path.parabola;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

import com.example.a18.path.R;

public class ParabolaActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_parabola);
        findViewById(R.id.pv).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

                ((ParabolaView) v).start();
                System.out.println("start");
            }
        });
    }
}

File: m_parabola/src/main/java/pdog18/com/module_parabola/ParabolaActivity.kt
package pdog18.com.module_parabola

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import android.view.View

class ParabolaActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_parabola)
        findViewById<View>(R.id.pv).setOnClickListener { v ->
            (v as ParabolaView).start()
            println("start")
        }
    }
}
